### PR TITLE
:bug: Implement clear notifications 

### PR DESCRIPTION
**Pull request by ibolton336**
https://issues.redhat.com/browse/MTA-28

### CODE REVIEW

File: client/src/app/shared/notifications-context.tsx [hunk 1]
```diff
@@ -34,14 +34,15 @@ export const NotificationsProvider: React.FunctionComponent<
...
-    }
+  const pushNotification = (
+    notification: INotification,
+    clearNotificationDelay?: number
+  ) => {
+    setNotifications([
+      ...notifications,
+      { ...notificationDefault, ...notification },
+    ]);
+    setTimeout(() => setNotifications([]), clearNotificationDelay || 10000);
```
Comment (gildub) [inline client/src/app/shared/notifications-context.tsx:45]: Can we replace hard coded value with a const store at higher level ?

**Review by gildub — CHANGES_REQUESTED** [review]

File: client/src/app/shared/notifications-context.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/shared/notifications-context.tsx:45]: Any particular reason?

Comment (gildub) [reply client/src/app/shared/notifications-context.tsx:45]: For me a hard coded value deep in the application is no go.
But ultimately is if we want the user to have control over it through an option then we'll have to lift it up.

Comment (gildub) [reply client/src/app/shared/notifications-context.tsx:45]: Never mind my comment, I totally misread it :(

**Review by gildub — APPROVED** [review]
LGTM

**Pull request merged**